/usr/bin/xvfb-run: line 173: kill: (34381) - No such process
/usr/bin/xvfb-run: line 173: kill: (34468) - No such process
/usr/bin/xvfb-run: line 173: kill: (35763) - No such process
/usr/bin/xvfb-run: line 173: kill: (34984) - No such process
/usr/bin/xvfb-run: line 173: kill: (34336) - No such process
/usr/bin/xvfb-run: line 173: kill: (35457) - No such process
/usr/bin/xvfb-run: line 173: kill: (40047) - No such process
usage: traj-cyc-env-adv.py [-h] rdis
traj-cyc-env-adv.py: error: the following arguments are required: rdis
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
982.0
532.0
51.0
982.0
532.0
51.0
0.3721721375247866 0.3988568244804811 0.2289710379947323 2969
anomalies
0.11858045442573029 0.5615653094459654 0.31985423612830455 0.5629151006951042
Traceback (most recent call last):
  File "traj-cyc-env-adv.py", line 332, in <module>
    cdata.to_csv('/atmosdyn2/ascherrmann/009-ERA-5/MED/traj/pandas-all-data.csv',index=False)
NameError: name 'cdata' is not defined
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
0.0
0.0
0.0
0.0
0.0
0.0
0.36788904325029803 0.40391284158226826 0.22819811516743374 4179
anomalies
0.059631220393339936 0.6417183515642092 0.29865042804245073 0.0
  File "traj-cyc-env-adv.py", line 178
    if(np.any(kickids==int(k)):
                              ^
SyntaxError: invalid syntax
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
0.0
0.0
0.0
0.0
0.0
0.0
0.36450840845030075 0.5321623347407392 0.10332925680895977 2414
anomalies
0.030297370909933827 0.4575174891678176 0.0898352954618514 0.0
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
0.0
0.0
0.0
0.0
0.0
0.0
0.36666827845451067 0.40913343137984975 0.2241982901656391 4098
anomalies
0.05619637966660489 0.6361521967627518 0.2882687961478149 0.0
Traceback (most recent call last):
  File "traj-cyc-env-adv.py", line 250, in <module>
    counterc60+=1
NameError: name 'counterc60' is not defined
Traceback (most recent call last):
  File "traj-cyc-env-adv.py", line 250, in <module>
    counterc60+=1
NameError: name 'counterc60' is not defined
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
1267.0
248.0
42.0
0.0
0.0
0.0
0.36450840845030075 0.5321623347407392 0.10332925680895977 2414
anomalies
0.030297370909933827 0.4575174891678176 0.0898352954618514 0.0
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
911.0
104.0
25.0
0.0
0.0
0.0
0.36819407196389753 0.5324316105148988 0.09937431752120378 1754
anomalies
0.05529484385426671 0.45675291394577916 0.07872354566913561 0.0
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
976.0
503.0
43.0
0.0
0.0
0.0
0.37099226720362344 0.404249210614328 0.2247585221820485 2888
anomalies
0.0997893278939614 0.5585096135172438 0.31441914548674016 0.0
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
1556.0
794.0
73.0
0.0
0.0
0.0
0.36666827845451067 0.40913343137984975 0.2241982901656391 4098
anomalies
0.05619637966660489 0.6361521967627518 0.2882687961478149 0.0
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
917.0
114.0
31.0
0.0
0.0
0.0
0.37064594829198977 0.5263668349009364 0.1029872168070739 1802
anomalies
0.05899882142025947 0.46317467031747767 0.08476487134747669 0.0
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
917.0
114.0
31.0
0.0
0.0
0.0
0.37064594829198977 0.5263668349009364 0.1029872168070739 1802
anomalies
0.11246997201209785 0.7435005116378116 0.14402951635009073 0.5607753823842796
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
911.0
104.0
25.0
0.0
0.0
0.0
0.36819407196389753 0.5324316105148988 0.09937431752120378 1754
anomalies
0.10883034405606508 0.7522816696039688 0.13888798633996624 0.5566839996724225
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
976.0
503.0
43.0
0.0
0.0
0.0
0.37099226720362344 0.404249210614328 0.2247585221820485 2888
anomalies
0.11668599077341499 0.569346355415056 0.3139676538115294 0.560976927167344
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
799.0
104.0
25.0
0.0
0.0
0.0
0.3698138924294471 0.5244743583578735 0.10571174921267952 1583
anomalies
0.11220464528771429 0.7400351165671842 0.14776023814510142 0.5585722397540769
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
864.0
503.0
43.0
0.0
0.0
0.0
0.3721525156546049 0.39071611603424183 0.2371313683111533 2712
anomalies
0.11929483479162041 0.5494795123127265 0.3312256528956529 0.5623839103073891
qt.qpa.xcb: could not connect to display localhost:19.0
qt.qpa.plugin: Could not load the Qt platform plugin "xcb" in "" even though it was found.
This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, minimal, minimalegl, offscreen, vnc, webgl, xcb.

usage: traj-cyc-env-adv.py [-h] rdis
traj-cyc-env-adv.py: error: the following arguments are required: rdis
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/fromnumeric.py:3335: RuntimeWarning: Mean of empty slice.
  out=out, **kwargs)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/numpy/core/_methods.py:161: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:1366: UserWarning: No contour levels were found within the data range.
  result = matplotlib.axes.Axes.contour(self, *args, **kwargs)
2969 2969
1613.0
1105.0
206.0
0.0
0.0
0.0
0.3721721375247866 0.3988568244804811 0.2289710379947323 2969
anomalies
0.11858045442573029 0.5615653094459654 0.31985423612830455 0.5629151006951042
numbers
1613 1105 206
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
composite-vcross.py:181: RuntimeWarning: invalid value encountered in true_divide
  compovcross_PV = PVn/counter
composite-vcross.py:181: RuntimeWarning: invalid value encountered in true_divide
  compovcross_PV = PVn/counter
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:782: MatplotlibDeprecationWarning: Passing the minor parameter of set_xticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_xticks(xticks, minor)
/usr/local/Miniconda3-envs/envs/2020/envs/iacpy3_2020/lib/python3.7/site-packages/cartopy/mpl/geoaxes.py:829: MatplotlibDeprecationWarning: Passing the minor parameter of set_yticks() positionally is deprecated since Matplotlib 3.2; the parameter will become keyword-only two minor releases later.
  return super(GeoAxes, self).set_yticks(yticks, minor)
